stm32/src/si53xx.c

```diff
@@ -42,6 +42,7 @@
 #include "math.h"
 #include "si53xx.h"
 
+T_Si5351_data   Si5351_Config;
 
```

Comment:
As you suggested on the mailing list, I'm OK simply rm-ing this and other sm2000 specific code.  It will still be there in the Git basement if we ever want it.